governator-jetty/src/main/java/com/netflix/governator/guice/jetty/Archaius2JettyConfig.java

```diff
@@ -34,4 +34,12 @@
      */
     @DefaultValue("/")
     String getWebAppContextPath();
+
+    @DefaultValue("true")
+    boolean isPlaintextSocketEnabled();
```

Comment:
plain text is a content type, but this boolean controls whether the socket uses encryption?  c/b isUnencryptedSocketEnabled()?

Reply:
:+1: